src/profile-logic/import/linux-perf.js

```diff
@@ -121,12 +121,21 @@ export function convertPerfScriptProfile(
         frame = frameTable.data.length;
         const location = stringTable.length;
         stringTable.push(frameString);
-        // Heuristic: 'kallsyms' presence in a frame seems to be a reasonably
-        // reliable indicator of a Linux kernel frame, and easier/more portable
+        // Heuristic: Treat frames which match any of the following conditions
+        // as Linux kernel frames:
+        // - contains kallsyms
+        // - ends in '/vmlinux' (seen when full kernel debug symbols are present,
```

Comment:
I think sometimes the `vmlinux` path has a version after it, e.g. `/usr/lib/debug/boot/vmlinux-4.16.0-1-amd64` , so it might be worth checking for a `/vmlinux` substring

Reply:
Ack.